molgenis/molgenis-emx2
# feat: filter variables on cohort variable mapping

Add the ability to filter (mapped) variables by cohort

Closes #3193

- Split list filter into list and filter part
- Move list to component lib, and add demo page
- Add cohort filter to variables page default filter config
- Add option to pass custom filterCondition build function to Filter type
- If cohort filter is active, filter the list by including only variables that either have a direct mapping to the cohort or have a repeat that has a mapping to the cohort.

how to test:
- go to the filters page
- select the cohorts filter -> cohorts should be loaded and shown
- select one or more cohorts 
- variables should now be filters to only include those that have mapping ( source) to the selected cohorts 

todo:
- [ ] updated testplan to include a test for this fix, including ref to bug using # notation
